pkgs/data/themes/sweet-nova/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, gitUpdater
+,
+}:
+stdenv.mkDerivation rec {
+  pname = "sweet-nova";
+  version = "unstable-2023-03-16";
```

Comment:
There is one from 2023-03-26 in the repo.
Indeed it is the latest one :) You wrote the date wrong.